qp_shotgun/humann2/tests/test_humann2.py

```diff
@@ -132,8 +136,11 @@ def test_generate_humann2_analysis_commands_forward_reverse(self):
             f.write(MAPPING_FILE)
         self._clean_up_files.append(fp)
 
+        out_dir = mkdtemp()
```

Comment:
Do we want to make this mapping file in setUp and include fp as a test class variable?